Sifchain/sifnode
# [Frontend] Feature/background events

* Background refresh pool balances for swap

To test:

Terminal 1: `yarn chain:sif`
Terminal 2: `yarn chain:eth`
Terminal 3: `yarn chain:migrate && yarn core:watch`
Terminal 4: `yarn app:serve`

Then launch the app http://localhost:8080

Setup a swap:
1. Connect wallet
1. Select tokens and amounts

Once up try to remove liquidity from a pool:

```bash
sifnodecli tx clp remove-liquidity \
 --from akasha \
 --sourceChain ETH \
 --symbol ETH \
 --ticker cbtk \
 --wBasis 9901 \
 --asymmetry -1
```

Notice the price output will change in accordance with the new pool amount from the swap.
